docs/feature_flags.md

```diff
@@ -239,9 +239,9 @@ Examples of equivalent durations:
 
 `--enable-feature=otlp-native-delta-ingestion`
 
-When enabled, allows for the native ingestion of delta OTLP metrics, storing the raw sample values without conversion. This cannot be enabled in conjunction with `otlp-deltatocumulative`.
+When enabled, allows for the native ingestion of delta OTLP metrics, storing the raw sample values without conversion. This cannot be enabled in conjunction with `otlp-deltatocumulative`. It is recommended to enable `type-and-unit-labels`.
```

Comment:
```suggestion
When enabled, allows for the native ingestion of delta OTLP metrics, storing the raw sample values without conversion, as gauge/gaugehistogram, as proposed in PROM-48 first milestone. 

> Long term, as per PROM-48, we plan to a new feature flag/mode where deltas will be ingested as counters with temporality label.

This feature cannot be enabled in conjunction with `otlp-deltatocumulative` feature flag. It is also recommended to enable `type-and-unit-labels`.
```

Reply:
Can we force use of `It is recommended to enable `type-and-unit-labels`? Wouldn't that be better UX? I think we should do this by default one day too (type and unit)